svc-licenses/licenses/licenses_test.go

```diff
@@ -92,3 +92,87 @@ func TestGetLicenseResource(t *testing.T) {
 	assert.Equal(t, int(response.StatusCode), http.StatusOK, "Status code should be StatusOK.")
 	assert.Equal(t, licenseResourceResponse, license, "Valid License resource response is expected")
 }
+
+func TestInstallLicenseService(t *testing.T) {
+	req := &licenseproto.InstallLicenseRequest{
+		RequestBody: []byte(`{
+			"LicenseString": "333K2-8QLV4-Y8R3G-LJQQX-7BK6M",
```

Comment:
is this the dummy license string?

Reply:
As of now i have kept string as XYZ, will change it to "XXXX-XXXX-XXXX-XXXX-XXXX" in my next PR for aggregates.